frontend/src/metabase/actions/containers/ActionCreator/ActionCreator.unit.spec.tsx

```diff
@@ -248,17 +281,15 @@ describe("ActionCreator", () => {
         userEvent.click(
           screen.getByRole("button", { name: "Action settings" }),
         );
-        expect(
-          screen.getByRole("textbox", { name: "Success message" }),
-        ).toHaveValue("Thanks for your submission.");
 
-        userEvent.type(
-          screen.getByRole("textbox", { name: "Success message" }),
-          `${specialChars.selectAll}Thanks!`,
-        );
-        expect(
-          screen.getByRole("textbox", { name: "Success message" }),
-        ).toHaveValue("Thanks!");
+        const messageBox = screen.getByRole("textbox", {
+          name: "Success message",
+        });
+        expect(messageBox).toHaveValue("Thanks for your submission.");
+
+        await waitFor(() => expect(messageBox).toBeEnabled());
+        userEvent.type(messageBox, `${specialChars.selectAll}Thanks!`);
+        expect(messageBox).toHaveValue("Thanks!");
```

Comment:
Need to `waitFor` until the action editor fetches the model, so it can see `can_write: true` and enable the textbox